turbinia/config/__init__.py

```diff
@@ -22,16 +22,23 @@
 import os
 import sys
 
+from turbinia import TurbiniaException
+
 DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
 
 # Look for config files with these names
-CONFIGFILES = ['.turbiniarc', 'turbinia.conf', 'turbinia_config.py']
-# Look in homedir first, then /etc/turbinia, and finally in the source
-# config dir for config files
+CONFIGFILES = ['.turbiniarc', 'turbinia.conf', 'turbinia_config_tmpl.py']
+# Look in homedir first, then /etc/turbinia
 CONFIGPATH = [
-    os.path.expanduser('~'), '/etc/turbinia',
-    os.path.dirname(os.path.abspath(__file__))
+    os.path.expanduser('~'),
+    '/etc/turbinia',
+    os.path.dirname(os.path.abspath(__file__)),
 ]
+# Config setup reminder for cleaner error handling on empty configs.
+# TODO(johngalvin): Link out to install guide anchors instead?
+CONFIG_MSG = (
+    'Copy turbinia/config/turbinia_config_tmpl.py to ~/.turbiniarc, '
```

Comment:
Maybe add `/etc/turbinia/turbinia.conf` to this as well?

Reply:
Done.